app/modules/individuals/resources.py

```diff
@@ -364,3 +364,121 @@ def get(self, individual):
+
+
+@api.route('/<uuid:individual_guid>/merge')
+@api.response(
+    code=HTTPStatus.NOT_FOUND,
+    description='Individual not found.',
+)
+@api.resolve_object_by_model(Individual, 'individual')
+class IndividualByIDMerge(Resource):
+    """
+    Merge from a list other Individual(s)
+    """
+
+    # permission on target individual is not needed, as user may only have access to other from_individuals
+    # @api.permission_required(
+    #     permissions.ObjectAccessPermission,
+    #     kwargs_on_request=lambda kwargs: {
+    #         'obj': kwargs['individual'],
+    #         'action': AccessOperation.READ,  # only requiring READ here as non-writers basically "request" a merge
+    #     },
+    # )
+    def post(self, individual):
+        from flask_login import current_user
+
+        if not current_user or current_user.is_anonymous:
+            abort(code=401)  # anonymous cannot even request a merge
+
+        req = json.loads(request.data)
+        from_individual_ids = req  # assume passed just list, check for otherwise
+        if isinstance(req, dict):
+            from_individual_ids = req.get('fromIndividualIds', None)
+        if not isinstance(from_individual_ids, list):
+            abort(
+                success=False,
+                message='must pass a list of individuals IDs to merge from',
+                code=500,
+            )
+        if len(from_individual_ids) < 1:
+            abort(
+                success=False,
+                message='list of individuals IDs to merge from cannot be empty',
+                code=500,
+            )
+
+        # NOTE when merge conflict (DEX-514) is addressed, more potential args will be passed in
+
+        meets_minimum = False
+        # for which user does not have edit permissions
+        blocking_encounters = []
+        from_individuals = []
+        for from_id in from_individual_ids:
+            from_indiv = Individual.query.get(from_id)
+            if not from_indiv:
+                abort(
+                    success=False,
+                    message=f'passed from individual id={from_id} is invalid',
+                    code=500,
+                )
+            for enc in from_indiv.encounters:
+                if enc.current_user_has_edit_permission():
+                    meets_minimum = True
+                else:
+                    blocking_encounters.append(enc)
+            from_individuals.append(from_indiv)
+        for enc in individual.encounters:
+            if enc.current_user_has_edit_permission():
+                meets_minimum = True
+            else:
+                blocking_encounters.append(enc)
+
+        if not meets_minimum:
+            AuditLog.security_alert(
+                log,
+                f'requested unauthorized merge from {from_individuals}',
+                individual,
+            )
+            abort(code=403)
+
+        # all is in order for merge; is it immediate or just a request?
```

Comment:
How can the merge be in order if there are blocking encounters?

Reply:
it becomes a merge **request**.   there are two ways merges can happen:
1. automatically (which requires edit-privileges on **all encounters**)
2. requested (user has _at least one_ encounter edit-privilege)

this is all in a related google doc; will hit you up on slack about this.........